repositories/src/main/scala/cromwell/pipeline/datastorage/dto/Project.scala

```diff
@@ -8,7 +8,7 @@ import play.api.libs.json.JsonNaming.SnakeCase
 import play.api.libs.json._
 import java.nio.file.{ Path, Paths }
 import slick.lifted.MappedTo
-
+// scalastyle:off number.of.types
```

Comment:
please create a ticket to split this file

Reply:
[Ticket](https://jirapct.epam.com/jira/browse/EPMLSTRCMW-303) created